docs/src/apps/jsonrpc-api.md

```diff
@@ -111,11 +111,14 @@ Requests can be sent in batches by sending an array of JSON-RPC request objects
 Solana nodes choose which bank state to query based on a commitment requirement
 set by the client. Clients may specify either:
 
-- `"max"` - the node will query the most recent bank confirmed by the cluster as having reached maximum lockout.
-- `"root"` - the node will query the most recent bank having reached maximum lockout on this node.
-- `"single"` - the node will query the most recent bank having reached 1 cluster confirmation.
-- `"singleGossip"` - the node will query the most recent bank having reached 1 cluster confirmation via gossip votes; may occur before or after `single`, depending on gossip traffic.
-- `"recent"` - the node will query its most recent bank.
+- `"max"` - the node will query the most recent bank confirmed by the cluster as having reached `MAX_LOCKOUT_HISTORY` confirmations
+- `"root"` - the node will query the most recent bank having reached `MAX_LOCKOUT_HISTORY` confirmations on this node
+- `"single"` - the node will query the most recent bank having reached 1 confirmation by supermajority of the network.
```

Comment:
Re: use of "bank" here -- I feel we should use "block" instead as this information is more intended to be outward facing and we don't define "bank" at https://docs.solana.com/terminology